packages/web_benchmarks/lib/src/browser.dart

```diff
@@ -23,15 +22,15 @@ class ChromeOptions {
     this.url,
```

Comment:
Please remove the comment several lines above about which things must not be null; the types now express that, so it's redundant.